flex/views/experiment/TabAnnotationView.mxml

```diff
@@ -480,7 +499,7 @@
 			selectedProperties = new XMLListCollection();
 			for each (var property:Object in filteredProperties) {
 				if (property.@isSelected == "true") {
-					selectedProperties.addItem(property);
+					selectedProperties.addItem(property.copy());
```

Comment:
Thanks for fixing this Matt.  Was adding a copy instead of the actual element a big cause of the non unique error?

If so, I have been burned by this very thing in the past.